kaggle-classification/keras_hparam_config.yaml

```diff
@@ -9,3 +9,28 @@ trainingInput:
   # parameterServerType: large_model
   # workerCount: 9
   # parameterServerCount: 3
+  hyperparameters:
```

Comment:
Can you add a comment indicating roughly how many versions of the model will be run with this config? It'd be nice to know if I'm setting off 10 jobs or 100 jobs or 1000 jobs.

Reply:
Do the maxTrials / maxParallelTrials above answer that?

Reply:
Got it. Thanks!